src/main/java/seedu/duke/data/notebook/Tag.java

```diff
@@ -96,4 +96,25 @@ public void setTagAttribute(Attribute tagColor) {
     public String toString() {
         return colorize("[" + tagName + "]", tagAttribute);
     }
+
+    public String toSaveString() {
+        String colorString = "";
+
+        if (tagAttribute.toString().equals(Attribute.BRIGHT_RED_TEXT().toString())) {
+            colorString = COLOR_RED_STRING;
+        } else if (tagAttribute.toString().equals(Attribute.BRIGHT_GREEN_TEXT().toString())) {
+            colorString = COLOR_GREEN_STRING;
+        } else if (tagAttribute.toString().equals(Attribute.BRIGHT_BLUE_TEXT().toString())) {
+            colorString = COLOR_BLUE_STRING;
+        } else if (tagAttribute.toString().equals(Attribute.BRIGHT_YELLOW_TEXT().toString())) {
+            colorString = COLOR_YELLOW_STRING;
+        } else if (tagAttribute.toString().equals(Attribute.BRIGHT_CYAN_TEXT().toString())) {
+            colorString = COLOR_CYAN_STRING;
+        } else if (tagAttribute.toString().equals(Attribute.BRIGHT_MAGENTA_TEXT().toString())) {
+            colorString = COLOR_MAGENTA_STRING;
+        } else if (tagAttribute.toString().equals(Attribute.BRIGHT_WHITE_TEXT().toString())) {
+            colorString = COLOR_WHITE_STRING;
```

Comment:
Can we use a switch statement?

Reply:
I tried but Attribute.COLOUR_TEXT().toString() isn't a constant. So it kept giving an error

Reply:
This is a temporary suggestion I provided her. Attritbute.RIGHT_COLOR_TEXT().toString cannot be used as case expression because they are not constant. There is another way to use switch, but since its related to color, I will edit on my side.